### PR TITLE
Project popup

### PR DESCRIPTION
**Pull request by JohnteFresh**
Add a mobile popup and styled it to work on bigger screens. 
Store My projects area work in JavaScript object
fix linter errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,132 +45,38 @@ <h1>Johnson Macharia</h1>
...
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
+    <section id="project-container" class="">
+      <h2>Projects</h2>
```
Comment (BregornOriginal) [inline index.html:54]: - [x] Kindly match with the Figma design, the title and list of skills should be aligned to center in the mobile version
- [x] Review the font size in this section, seems to be a bit bigger.

image

File: index.html [hunk 2]
```diff
@@ -47,132 +45,38 @@ <h1>Johnson Macharia</h1>
...
+      <div class="dec-2"><img class="decoration" src="icons/image-geometry_2 1.png"></div>
+      <div class="dec-3"><img class="decoration" src="icons/image-geometry_3.png"></div>
```
Comment (BregornOriginal) [inline index.html:56]: - [x] Kindly add icons missing in your buttons 

image

- [x] Kinldy add the different hover effects to the buttons as it's on the Figma design

image

File: index.html [hunk 3]
```diff
@@ -47,132 +45,38 @@ <h1>Johnson Macharia</h1>
...
+      <div class="dec-4"><img class="decoration" src="icons/image-geometry_4.png"></div>
+      <div class="dec-5"><img class="decoration" src="icons/image-geometry_5.png"></div>
```
Comment (BregornOriginal) [inline index.html:58]: - [x] In your desktop version, add the icons missing on the buttons
- [x] Add the arrows missing in your `Previous project` and `Next project` buttons

image

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @Team

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (4)

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Excellent work implementing the Popup window! 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
## Hi @JohnteFresh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- UI design looking awesome



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**